templates/traefik-forward-auth.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: traefik-forward-auth
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.14.0
+  chartReference:
+    chart: https://github.com/jongiddy/traefik-forward-auth-chart/archive/oidc-issuer.tar.gz
```

Comment:
As below, could we get a PR with the chart in `mesosphere/charts` ?

Reply:
This should be faster. I want to remove the last vestiges of Konvoy-specific elements from the chart (the references to "well-known" secrets) and then I will add a PR to `mesosphere/charts`.

Reply:
Since this chart is very specific to the Konvoy setup, with secrets obtained from other components, is it perhaps better to replace the Helm chart with a Kubernetes spec?

Reply:
We can go for now with this option 😞 to get this IN. But please, create a PR to mesosphere/charts so we can do the changes there.

Reply:
https://github.com/mesosphere/charts/pull/16

Reply:
Thanks